specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,861 @@
+          },
+          "404": {
+            "description": "Container Apps Job does not exist.",
+            "x-ms-error-response": true
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Container Apps Job": {
+            "$ref": "./examples/ContainerAppsJob_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      },
+      "patch": {
+        "tags": [
+          "ContainerAppsJobs"
+        ],
+        "summary": "Update properties of a Container Apps Job",
+        "description": "Patches a Container Apps Job using JSON Merge Patch",
+        "operationId": "ContainerAppsJob_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppsJobName",
+            "in": "path",
+            "description": "Name of the Container Apps Job.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "containerAppsJobEnvelope",
+            "in": "body",
+            "required": true,
+            "description": "Properties used to create a container apps job",
+            "schema": {
+              "$ref": "#/definitions/ContainerAppsJobPatchProperties"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/ContainerAppsJob"
+            }
+          },
+          "202": {
+            "description": "Patch operation is in progress.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Patch Container Apps Job": {
+            "$ref": "./examples/ContainerAppsJob_Patch.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerAppsJobs/{containerAppsJobName}/run": {
```

Comment:
>run [](http://example.com/codeflow?start=137&length=3)

I think start and cancel are better action names for this and the next POST action. Just offering an opinion, this wouldn't block ARM signoff.

Reply:
Agree. Should we call this start and rename "terminate" to "stop"

Reply:
@vinisoto please suggest. Currently we have run and terminate as the actions.

Reply:
I'm ok with the change.

Reply:
@mentat9 thanks for the suggestion. Updated this.